SparkPost/2web2ui
# SA-1997: Add Industry Benchmarking to charts

### What Changed

- Added tooltip section for industry benchmark
- Added average in aggregate section
- Added area in chart for industry benchmark

### How To Test

- Go to appteam account
- Enable inbox folder rate metric
- Verify that the inbox folder rate appears

### To Do

- [ ] Add integration testing?
- [ ] Add unit testing?

### Notes
- A bit hard to do unit testing around recharts
- Integration testing a bit difficult as well until the parts... integrated (SA-1993)
